src/lib/y2security/security_policies/manager.rb

```diff
@@ -0,0 +1,140 @@
+require "y2security/security_policies/issue"
+
+module Y2Security
+  module SecurityPolicies
+    # Class to manage security policies
+    class Manager
+      include Singleton
+
+      # Environment variable to enable security policies
+      #
+      # Its value contains "comma-separate" ids of the policies to enable
+      ENV_SECURITY_POLICIES = "YAST_SECURITY_POLICIES".freeze
+      private_constant :ENV_SECURITY_POLICIES
+
+      # Currently enabled policies
+      #
+      # @return [Array<Policy>]
+      attr_reader :enabled_policies
+
+      # Policies are automatically enabled according the the environment variable
+      # ENV_SECURITY_POLICIES
+      def initialize
+        @enabled_policies = []
+
+        enable_policies
+      end
+
+      # Returns the list of known security policies
+      #
+      # @return [Array<Policy>]
+      def policies
+        @policies ||= [DisaStigPolicy.new].freeze
+      end
+
+      # Finds the security policy with the given id
+      #
+      # @param id [Symbol] Security policy id
+      # @return [Policy, nil]
+      def find_policy(id)
+        policies.find { |p| p.is?(id) }
+      end
+
+      # Enables the given policy
+      #
+      # @param policy [Policy]
+      def enable_policy(policy)
+        return unless policies.include?(policy)
+
+        @enabled_policies.push(policy).uniq!
+      end
+
+      # Disables the given policy
+      #
+      # @param policy [Policy]
+      def disable_policy(policy)
+        @enabled_policies.delete(policy)
+      end
+
+      # Whether the given policy is enabled
+      #
+      # @param policy [Policy]
+      # @return [Boolean]
+      def enabled_policy?(policy)
+        @enabled_policies.include?(policy)
+      end
+
+      # Issues from all enabled policies
+      #
+      # The issues can be limited to a specific scope.
+      #
+      # @param scope [Scopes::Storage, Scopes::Bootloader, Scopes::Network, Scopes::Firewall]
+      # @return [IssuesCollection]
+      def issues(scope = nil)
+        issues_collection = IssuesCollection.new
+
+        enabled_policies.each do |policy|
+          issues_collection.update(policy, policy.validate(scope))
+        end
+
+        issues_collection
+      end
+
+    private
+
+      # Enables policies according to the environment variable ENV_SECURITY_POLICIES
+      def enable_policies
+        policies_from_env.each { |p| enable_policy(p) }
+      end
+
+      # Policies from the values indicated with the environment variable ENV_SECURITY_POLICIES
+      #
+      # @return [Array<Policy>]
+      def policies_from_env
+        env_policies.map { |v| policy_from_env(v) }.compact.uniq
+      end
+
+      # Policy from one of the values indicated with the environment variable ENV_SECURITY_POLICIES
+      #
+      # @param value [String]
+      # @return [Policy, nil]
+      def policy_from_env(value)
+        return find_policy(:disa_stig) if value.match?(/\Adisa_stig\z/i)
```

Comment:
So only `disa_stig` is supported, right?

Reply:
Yes.